### PR TITLE
DOC: added examples to DataFrame.std

### PR DESCRIPTION
**Pull request by ruankie**
xref https://github.com/pandas-dev/pandas/issues/44162
- [x] Added two examples to the documentation of DataFrame.std function so that users can understand how to use it with different delta degrees of freedom.
- [ ] Add examples to the documentation of DataFrame.var function
---
- [x] tests added / passed
- [x] All pre-commit linting tests pass

### CODE REVIEW

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]
Thanks @ruankie 

~Content looks good, you just need to fix up linting, see https://pandas.pydata.org/docs/development/contributing_documentation.html#updating-a-pandas-docstring~

Can you address https://github.com/pandas-dev/pandas/pull/44167#discussion_r735177095 please?

**Comment by ruankie** [comment]
Finally managed to successfully build the docs locally with the examples inserted (please see attached screenshot)

std_examples

File: pandas/core/generic.py [hunk 1]
```diff
@@ -11185,6 +11190,34 @@ def _doc_params(cls):
...
+...                  ).set_index('person_id')
+>>> df
+           age  height
+person_id
+0           21    1.61
+1           25    1.87
+2           62    1.49
+3           43    2.01
+
+The standard deviation of the columns can be found as follows.
```
Comment (MarcoGorelli) [inline pandas/core/generic.py:11208]:
```suggestion
The standard deviation of the columns can be found as follows:
```

File: pandas/core/generic.py [hunk 2]
```diff
@@ -11185,6 +11190,34 @@ def _doc_params(cls):
...
+
+>>> df.std()
+age       18.786076
+height     0.237417
+
+Alternatively, `ddof=0` can be set to normalize by N instead of N-1.
```
Comment (MarcoGorelli) [inline pandas/core/generic.py:11214]:
```suggestion
Alternatively, `ddof=0` can be set to normalize by N instead of N-1:
```

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]
Good job, thanks! ~Couple of nitpicks, then good to go~ (EDIT: see futher review below)

File: pandas/core/generic.py [hunk 3]
```diff
@@ -11176,6 +11179,8 @@ def _doc_params(cls):
 -------
 {name1} or {name2} (if level specified) \
 {notes}
+
+{examples}
```
Comment (MarcoGorelli) [inline pandas/core/generic.py:11183]: Will there be extra whitespace at the end for e.g. `sem` (where you have `examples=""`) now? Could you post a screenshot of those docs too please?

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]

File: pandas/core/generic.py (hunk 3 above)
Comment (ruankie) [reply pandas/core/generic.py:11183]: Will double-check and send a screenshot shortly

**Comment by ruankie** [comment]
Just changed it to be more consistent with other function docstrings. I believe there are no extra whitespaces (as seen in screenshots)

---
`pandas.DataFrame.std`

std

---
`pandas.DataFrame.sem`

sem

**Review by MarcoGorelli — APPROVED** [review]
Nice, thanks!

**Comment by jreback** [comment]
thanks @ruankie 

happy to take a followup for .var to close this issue out.

**Pull request merged**